lib/modulr/resources/transfers/transfer.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Modulr
+  module Resources
+    module Transfers
+      class Transfer < Base
+        attr_accessor :id, :externalReference, :status # rubocop:disable Naming/MethodName
+
+        alias payment_reference_id id
+        alias external_reference externalReference
```

Comment:
Do NOT use the attr_accessor/alias pattern. Please replace it with the map helper:

```suggestion
        map :id, [:id, :payment_reference_id]
        map :status, :status
        map :externalReference, :external_reference
```